apps/desktop/src/app/sync/sources/brightcove.ts

```diff
@@ -9,15 +9,15 @@ import {
 } from '../models';
 import { Observable, of, from, map, switchMap } from 'rxjs';
 
-export const BrightcoveConnector: SourceConnectorDefinition = {
-  id: 'brightcove',
-  title: 'Brightcove',
-  logo: 'assets/logos/brightcove.svg',
-  description: 'Video delivery platform',
-  factory: () => of(new BrightcoveImpl()),
-};
+// export const BrightcoveConnector: SourceConnectorDefinition = {
+//   id: 'brightcove',
+//   title: 'Brightcove',
+//   logo: 'assets/logos/brightcove.svg',
+//   description: 'Video delivery platform',
+//   factory: () => of(new BrightcoveImpl()),
+// };
```

Comment:
dead code

Reply:
All the connectors but OneDrive and DropBox haven't been migrated to the new system, I want to keep the code because the way they use their expected API is relevant.